src/frontend/src/icp/utils/icp-transactions.utils.ts

```diff
@@ -56,11 +56,11 @@ export const mapIcpTransaction = ({
 	transaction: IcpTransaction;
 	identity: OptionIdentity;
 }): IcTransactionUi => {
-	const { operation, created_at_time, transferToSelf } = transaction;
+	const { operation, timestamp, transferToSelf } = transaction;
```

Comment:
Did you double check with FI team or are we sure the timestamp is a better source?

That said, already approving.

Reply:
As per ongoing discussion with FI, they confirmed the scope of timestamp (as you said); they still need to confirm what is the use-case of "missing" timestamp; but since it seems something not really on our control. So, to order the transactions, it seems ok